### ISSUE TITLE
USWDS v3 - Radio button - Accessibility review

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#1299]
## Issue Description
We need to conduct a review of the accessibility guidance on USWDS for the Radio buttons component  (https://designsystem.digital.gov/components/radio-buttons/) and compare it to the a11y guidance in the VA DS.  (https://design.va.gov/components/form/radio-button) 

## Tasks
- [x] Full review of accessibility guidance for the component on both USWDS and VA DS. 
- [x] Identify and document the differences between the two 
- [x] Make a recommendation on which guidance to use and include brief explanation on why (feedback can be in this ticket)
- [x] Review final recommendation with the team (can be done during standup) 
- [x] Add a11y guidance updates to the documentation ticket [Conversion to USWDS v3 - Radio Button - Documentation](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1269) (if needed) 

## Acceptance Criteria
- [ ] Differences of the a11y guidance for the component has been documented (can be in this ticket)
- [ ] Recommendation has been shared with the team
- [ ] Documentation ticket has been updated with the revised text for a11y guidance (if needed)

**Comment by SarahKay8** [comment]
Part 1: USDWS Radio Button - Findings 


1. The second “Fredrick Douglass” radio button includes additional descriptive text where it states “This is optional text that can be used to describe the label in more
          detail”.  
This is added to the label, rather than semantically as the radio button’s actual accessible description via `aria-describedby`.  They may have their reasons for this, but regardless, the text is also directly adjoined to the end of the label without any whitespace or punctuation that would make a screen reader pause.
2. The page includes disabled radio buttons.  disabled controls are confusing for many users and should often not be relied upon.

**Comment by caw310** [comment]
@humancompanion-usds please review Sarah's findings.

**Comment by humancompanion-usds** [comment]
Looks good to me. @SarahKay8 - Go ahead and make the updates to the page and assign to me for review. If you need help with that Natalie should be able to help or I can. For the tile option (description) that USWDS has. We just added that so work with @jamigibbs to make the aria-describedby change you described. This is a great example of where USWDS may not have thought of this and we can correct something before we give it back to them. 👍

**Comment by SarahKay8** [comment]
@humancompanion-usds will do! Thanks Matt
